### PR TITLE
Implemented the emailing service

### PR DESCRIPTION
**Pull request by veyselboybay**
Changed the emailing service to `SendGrid` from `Twilio`. Tested the service.

### CODE REVIEW

File: Server/utils/sendEmail.js [hunk 1]
```diff
@@ -1,43 +1,38 @@
...
-const logger = require('../utils/logger')
-
-
-const mailersend = new MailerSend({
-    apiKey: process.env.MAILERSEND_API_KEY,
-})
+const sgMail = require('@sendgrid/mail')
+const logger = require('./logger')
+const dotenv = require('dotenv')
+dotenv.config({path:__dirname+'/../.env'})
```
Comment (popnfresh234) [inline Server/utils/sendEmail.js:4]: I think this path is specifying .env should be in the root project folder?  If that's the case we should make a note in the readme to tell end user's where to put it.

I've been using the Server root folder so no config is necessary there.  Might be good to keep the server config and client config separate?

**Review by popnfresh234 — APPROVED** [review]
Looks good to me :+1:

File: Server/utils/sendEmail.js (hunk 1 above)
Comment (veyselboybay) [reply Server/utils/sendEmail.js:4]: I ran into an issue that it wasn't reading the .env file. I just figured I was testing it as a standalone, not inside the server. I will make the changes.

**Review by popnfresh234 — APPROVED** [review]

**Comment by popnfresh234** [comment]
Looks good to merge!

**Pull request merged**